VuChungDung/style.css

```diff
@@ -0,0 +1,106 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+    text-decoration: none;
+    list-style: none;
+    color: black;
+}
+.wrap{
+    width: 1024px;
+    margin: 0 auto;
+}
+.header{
+    width: 100%;
+    background-color: white;
+}
+.header-content{
+    padding: 0 40px;
+    height: 60px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.navbar-menu{
+    display: flex;
+}
+.navbar-menu__item{
+    width: 100px;
+    text-align: right;
+}
+.navbar-menu__link{
```

Comment:
Should make a commont css like .font-size-15 { font-size: 15px; } 
It's the good way for some class has only one common property like: padding, margin, font,....